src/styles/blocks/day.scss

```diff
@@ -0,0 +1,26 @@
+.calendar__day {
+  width: $day-size;
+  height: $day-size;
+  border: $border-size solid black;
+  background-color: $grey-color;
```

Comment:
Ensure that the `$grey-color` variable is defined. If not, this could lead to a compilation error or unexpected styling results.